src/pypi_attestations/_impl.py

```diff
@@ -331,3 +331,54 @@ def _ultranormalize_dist_filename(dist: str) -> str:
         return f"{name}-{ver}.tar.gz"
     else:
         raise ValueError(f"unknown distribution format: {dist}")
+
+
+class Publisher(BaseModel):
+    """Publisher as defined in PEP 740."""
+
+    kind: str
+    """
+    The kind of Trusted Publisher.
+    """
+
+    claims: object | None
```

Comment:
This should probably be:

```suggestion
    claims: dict[str, Any] | None
```

...since we expect `claims` to be a string-keyed mapping, not any object 🙂

Reply:
(I just realized the PEP lists it as `object`, but that was just because the PEP doesn't use Pydantic at all.)